### DISCUSSION TITLE
Pydantic V2 dependency set at <2.0.0 in pyproject.toml

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [fastapi/fastapi#10071] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
n/a
```

### Description

The pyproject.toml file sets Pydantic V2 dependency at <2.0.0 but latest release is v2.1.1. Does this matter?

https://github.com/tiangolo/fastapi/blob/efc12c5cdbede6922a033a5234c70cb22c4d204a/pyproject.toml

dependencies = [
    "starlette==0.22.0",
    "pydantic >=1.6.2,!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<2.0.0",
]

### Operating System

Windows

### FastAPI Version

0.101.0

### Pydantic Version

2.1.1

### Python Version

3.11.4

**Comment by jgould22** [comment]
The file you linked is from December of last year. the current one for version 101.0 is this https://github.com/tiangolo/fastapi/blob/0.101.0/pyproject.toml
